peekingduck/pipeline/nodes/input/live.py

```diff
@@ -56,6 +58,10 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
                         "pipeline_end": False,
                         "filename": self.filename,
                         "fps": self.fps_saved_output_video}
+            self.frame_counter += 1
```

Comment:
Here's an interesting point to ponder about - will integer overflow be a problem?
https://stackoverflow.com/questions/52151647/integer-overflow-in-python3

If not, will `self.frame_counter` become so big that it eats up a lot of memory?
https://code.tutsplus.com/tutorials/understand-how-much-memory-your-python-objects-use--cms-25609